langchain/src/chat_models/anthropic.ts

```diff
@@ -377,6 +377,7 @@ export class ChatAnthropic extends BaseChatModel implements AnthropicInput {
         ...this.clientOptions,
         ...options,
         apiKey: this.anthropicApiKey,
+        maxRetries: 0,
```

Comment:
this looks unrelated - was changing the Anthropic client intentional?

Reply:
Yes, oversight earlier but we handle retries natively through the `AsyncCaller` class for standardization.